apps/nextjs-website/src/app/page.tsx

```diff
@@ -30,8 +31,19 @@ const NotSsrWebinarsSection = dynamic(
   { ssr: false }
 );
 
+const NotSsrPastWebinarsShowcase = dynamic(
+  () =>
+    import('@/components/organisms/PastWebinarsShowcase/PastWebinarsShowcase'),
+  { ssr: false }
+);
+
 const Home = async () => {
   const webinars = await getVisibleInHomeWebinars();
+  const futureWebinars = getFutureWebinars(webinars);
+  const pastWebinars = getPastWebinars(webinars).slice(
+    0,
+    maxPastWebinarsInHome
+  );
```

Comment:
I think that if you filter the webinars here, it will be done at build time, so the dynamic rendering of the components will be useless.
Maybe you should render dynamically a component that's responsible for filtering the webinars and that wraps the following piece of code 
`      <NotSsrWebinarsSection
        title={
          futureWebinars.length > 1 ? 'dontLoseNextPlural' : 'dontLoseNext'
        }
        webinars={[...futureWebinars]}
      />
      <NotSsrPastWebinarsShowcase webinars={[...pastWebinars]} />`.

I would rename the `WebinarsSection` component as `FutureWebinarsShowcase` and name the new wrapper component as `WebinarsSection`